public/css/list/redundancy-group-list-item.less

```diff
@@ -1,6 +1,28 @@
+.item-list.detailed { //TODO: ipl-web item-list applies these changes for .default-layout class, which is removed for detailed view, because detail view can show long display names on multiple lines.
+  .redundancy-group-list-item { // but this margin rules are actually general and can be applied to all list items, so we should add them separately (without .default-layout class) in item-list.less instead to generalize them.
+    .title > * {
+      margin: 0 .28125em; // 0 calculated &nbsp; width
+
+      &:first-child {
+        margin-left: 0;
+      }
+
+      &:last-child {
+        margin-right: 0;
+      }
+    }
+  }
+}
+// remove code above once https://github.com/Icinga/ipl-web/pull/240 is merged.
```

Comment:
please clean this up, the linked pr is closed and the label seems still being wrapped in a `span`